### ISSUE TITLE
NMM 62.16 crash after uninstalling a mod

### ISSUE THREAD

**Issue by h-lucz** [Nexus-Mods/NMM-Test-Builds#104]
Mod. Gold Kit for Color Pipboy -anti-Black and White Screen--11063-1-11, Download ID confirmed.
Crash after uninstalling/deactivating (not completely removed). Has not happened again so far.
Trace Log:
[TraceLog20160526130605.txt](https://github.com/Nexus-Mods/NMM-Test-Builds/files/285310/TraceLog20160526130605.txt)

**Comment by KunoMochi** [comment]
Error message:

```
Tracing an Unhandled Exception:
                Exception: 
Message: 
    Der Prozess kann nicht auf die Datei "G:\Games\Nexus Mod Manager\Fallout4\Mods\ModProfiles\3i3hct5a\modlist.xml" zugreifen, da sie von einem anderen Prozess verwendet wird.
```

Roughly translates to

```
The process can not access the file "G:\Games\Nexus Mod Manager\Fallout4\mods\ModProfiles\3i3hct5a\modlist.xml" access, because it is used by another process.
```

This usually happens when one process is trying to edit a file that is already being edited and in use by another process.

Were you having NMM doing anything else while it was uninstalling/deactivating the mod?

**Comment by DuskDweller** [comment]
Does that happen just with that mod?

**Comment by h-lucz** [comment]
@KunoMochi , no, just uninstalled this mod for keeping an installation order. It happened only once. I afterwards installed and unintalled the mod repeatedly for testing order of installation ..., it has happened only once so far. So far, just for info to the NMM team.
@DuskDweller, sorry, for this mod once - for relating mods like DEF_UI, ICONLIBS2 etc. - no. Therefore  the Trace Log just for info ...
